docker/base/Dockerfile

```diff
@@ -8,7 +8,7 @@ COPY ./ /opt/labgrid/
 
 RUN set -e ;\
     apt update -q=2 ;\
-    apt install -q=2 --yes --no-install-recommends python3 python3-dev python3-pip python3-setuptools python3-wheel git build-essential libsnappy-dev ;\
+    apt install -q=2 --yes --no-install-recommends python3 python3-dev python3-pip python3-setuptools python3-wheel git build-essential libsnappy-dev libusb-1.0-0-dev libudev-dev ;\
```

Comment:
Consider moving the lib dependency to the images where it is needed.
As I understand your contribution this is relevant for client and exporter ?